content/articles/web-automation-with-selenium-in-python/index.md

```diff
@@ -0,0 +1,107 @@
+### Introduction
+Automation can be seen as a process of removing human effort in a process. It can be seen as a process of using electronic machines or robots to perform tasks. Automation can be done in many processes where humans are involved but in this article, we will be looking at automating web processes. 
+The ability to enable software robots to automatically perform processes and tasks on the web is known as WEB AUTOMATION. These processes may include; searching the web, deleting emails, filling forms, logging in to a website, and many other processes. The need for speed in performing tasks is a necessity in our modern world, this makes the automation of repetitive tasks necessary.
+Selenium is a framework for web application testing. In python, Selenium can be seen as a set of libraries that helps developers interact with the web to enable the automation of web processes. Primarily Selenium is used for automating software tests but it is also used for web scraping. Selenium is a very powerful tool when it comes to interacting with web browsers, it works with all modern web browsers and its scripts can be written in different programming languages such as Java, python, c#, etc.
+In this guide, we will be looking at how to use selenium to write scripts that will automate basic web tasks using python to give you a background of how Selenium works.
```

Comment:
`automate basic web tasks`

Can I automate my bank payments? Can I solve the CAPTCHA  automatically? (Just kidding, no offense)

Please be specific on what you write